back/src/pods/room/baseTrainerTokens.ts

```diff
@@ -323,19 +323,3 @@ const names = [
   'yuuto',
   'yuzuki',
 ];
-
-const formatToUpperCase = (name: string): string => {
```

Comment:
Why remove this functionallity?

Reply:
Because both trainer tokens and room names are lowercase in code-paster.

Moreover, that function was not working properly anyway.